html/html-and-the-web_div-salad/index.html

```diff
@@ -13,45 +13,46 @@
   </head>
   <!--someone forgot to wrap the content of our webpage with an HTML element,
 	which one was it again?-->
-  <div>
-    <div>
-      <div>Jane Doe</div>
+  <body>
+    <header>
+      <h2>Jane Doe</h2>
       <!--Which elements could be used to link to parts of the webpage,
 		which appear below?-->
-      <div id="nav">
-        <div>About Me</div>
-        <div>Contact</div>
-        <div>Skillset</div>
-      </div>
-    </div>
+      <nav id="nav">
+        <a href="About Me">About Me</a>
+        <a href="Contact">Contact</a></div>
+        <a href="Skillset">Skillset</a></div>
+      </nav>
+    </header>
     <!--Looks like the following part is the central content of our webpage.-->
-    <div>
+    
+  <div>
```

Comment:
I think you don't need this <div>. The next one with the id makes totally sense, but I think the code looks better without this one.